src/pages/api/upload-background.ts

```
@@ -0,0 +1,60 @@
+
+  try {
+    const [fields, files] = await form.parse(req)
+    const file = files.file?.[0]
+
+    if (!file) {
+      return res.status(400).json({ error: 'No file uploaded' })
+    }
+
+    const validExtensions = ['.jpg', '.jpeg', '.png', '.gif', '.webp']
+    const extension = path.extname(file.originalFilename || '').toLowerCase()
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)